templates/compose/ollama-with-open-webui.yaml

```diff
@@ -0,0 +1,47 @@
+# documentation: https://docs.openwebui.com
+# slogan: Ollama with Open Web UI integrates AI model deployment with a user-friendly interface.
+# tags: ollama,ai,models,deployment,open-web-ui,integration
+# logo: svgs/ollama.svg
+
+version: "3.9"
+
+services:
+  ollama-api:
```

Comment:
This seems to be the configuration for CPU inference. But the main power tool is the GPU inference. [Maybe you want to add the GPU devices?](https://docs.docker.com/compose/how-tos/gpu-support/#enabling-gpu-access-to-service-containers)

Reply:
The idea of this PR is to add only the base version of ollama running with support for CPU only, because according to ollama's documentation, when adding GPU support, the configuration varies between NVIDIA and AMD GPUs, for example, and I don't have all environments available for testing. I believe that with this CPU version, future contributors can add separate templates for AMD and NVIDIA GPU versions. The main idea is that this service can be run on most hosts where it is installed.